edit/buildozer.go

```diff
@@ -348,6 +348,31 @@ func cmdRemove(opts *Options, env CmdEnvironment) (*build.File, error) {
 	return nil, nil
 }
 
+func cmdRemoveComment(opts *Options, env CmdEnvironment) (*build.File, error) {
+	switch len(env.Args) {
+	case 0: // Remove comment attached to rule
+		env.Rule.Call.Comments.Before = nil
+		env.Rule.Call.Comments.Suffix = nil
+	case 1: // Remove comment attached to attr
+		if attr := env.Rule.AttrDefn(env.Args[0]); attr != nil {
+			attr.Comments.Before = nil
+			attr.Comments.Suffix = nil
+			attr.LHS.Comment().Before = nil
+			attr.RHS.Comment().Suffix = nil
+		}
+	case 2: // Remove comment attached to value
+		if attr := env.Rule.Attr(env.Args[0]); attr != nil {
+			if expr := ListFind(attr, env.Args[1], env.Pkg); expr != nil {
+				expr.Comments.Before = nil
+				expr.Comments.Suffix = nil
```

Comment:
The same as above (although arguments normally have just before or suffix comments, it's cleaner to remove everything).

Reply:
Done, additionally nil'd out every field on the attr and attr.LHS and attr.RHS.

Reply:
Good catch, a comment like ` attr = value  # comment` will actually be attached to the RHS, not to the assignment node.